tools/remote_build/rbe_common.bazelrc

```diff
@@ -43,6 +43,7 @@ build --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
 build --define GRPC_PORT_ISOLATED_RUNTIME=1
 # without verbose gRPC logs the test outputs are not very useful
 test --test_env=GRPC_VERBOSITY=debug
+test --test_env=GRPC_TEST_RUNNING_UNDER_RBE=1
```

Comment:
isn't there a better way of detecting this?
- I'd like to keep the .bazelrc configuration as small as possible (maintenance reasons)
- I'm afraid of the creep when over time more tests start relying on this ad-hoc env variable.